resolwe_bio/kb/migrations/0005_species.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-10-27 06:27
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('resolwe_bio_kb', '0004_add_unique_together'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='mapping',
+            name='source_species',
+            field=models.CharField(default='unknown', max_length=50),
```

Comment:
We could infer this based on existing features in the database instead of using unknown. But when this change will be merged, we will probably repopulate all mappings anyway, so this should not be a problem?

Reply:
Aren't there some features that doesn't have unique `(db, id)` combination? I think that it is safer (and easier) to repopulate mapings.

Reply:
Yes, this is correct, but the old database could not hold those anyway, because previously `(db, id)` were the primary key. I agree that it is safer to repopulate mappings, so we should leave it as `unknown` in this migration.